docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,542 @@
 .. _hpx_1_5_0:
 
 ===========================
-|hpx| V1.5.0 (TBD)
+|hpx| V1.5.0 (?)
 ===========================
 
 General changes
 ===============
 
+* We have added the C++20 features ``hpx::jthread`` and ``hpx::stop_token``. We
```

Comment:
We should add a couple of higher-level sentences describing the main focus of this release, i.e. C++20 conformance, modularization, performance improvements, API unification, etc.